Mage.Client/src/main/java/mage/client/dialog/NewTableDialog.form

```diff
@@ -364,6 +364,14 @@
     </Component>
     <Component class="javax.swing.JComboBox" name="cbTimeLimit">
     </Component>
+    <Component class="javax.swing.JLabel" name="lbBufferTime">
+      <Properties>
+        <Property name="text" type="java.lang.String" value="Buffer Time:"/>
+        <Property name="toolTipText" type="java.lang.String" value="The extra time a player gets whenever the timer starts before their normal time limit starts going down."/>
+      </Properties>
+    </Component>
+    <Component class="javax.swing.JComboBox" name="cbBufferTime">
```

Comment:
The NewTournamentDialog sets a tooltip here, but this NewTableDialog doesn't. Is this intentional or should one of them be changed?

Reply:
This is consistent with the prior implementation for the time limit option. I honestly don't think a combo box needs a tooltip - it makes more sense to hover the label.